### PR TITLE
Feature/fix latent heat and longwave coupler fields

### PR DESCRIPTION
**Pull request by dgergel**
This PR makes the following changes in the CESM driver: 

- fixes sign for latent heat flux passed from VIC to the coupler
- fixes sign for longwave radiation (lwup) passed from VIC to the coupler

### CODE REVIEW

File: vic/drivers/cesm/src/cesm_put_data.c [hunk 1]
```diff
@@ -258,7 +258,7 @@ vic_cesm_put_data()
```
Comment (jhamman) [inline vic/drivers/cesm/src/cesm_put_data.c]: Does VIC have different sign conventions for `AtmosLatent` and `AtmosSensible`?

Comment (dgergel) [inline vic/drivers/cesm/src/cesm_put_data.c]: They are the same in `~/vic/vic/drivers/shared_all/src/put_data.c`, but these sign conventions agree with those in VIC 4 in RASM (e.g. these make the coupler fields for VIC 5 in RASM agree with VIC 4 in RASM).

File: docs/Development/ReleaseNotes.md [hunk 2]
```diff
@@ -36,7 +38,12 @@ To check which release of VIC you are running:
 	7. Fixed bug that prevented using the correct local domain grid cells in `cesm_put_data.c`
 	8. Changed reference temperature units from Celsius to Kelvin in `cesm_put_data.c`
 
-	[GH#696](https://github.com/UW-Hydro/VIC/pull/696)
+  ([GH#695](https://github.com/UW-Hydro/VIC/pull/695))
+
+	1. Fix sign for latent heat fluxes passed from VIC to the coupler
+	2. Fix sign for longwave radiation passed from VIC to the coupler
+
+	([GH#696](https://github.com/UW-Hydro/VIC/pull/696))
```
Comment (jhamman) [inline docs/Development/ReleaseNotes.md]: there was a merge conflict here that I corrected via github, this does't look quite right though. Can you check/fix this.

Comment (dgergel) [inline docs/Development/ReleaseNotes.md]: Should be all fixed now.

**Pull request merged**